### PR TITLE
Update take.md

### PR DESCRIPTION
**Pull request by mixmix**
the `pull.take` were a bit sparse and hard to follow.
Here's an improvement

### CODE REVIEW

**Comment by mixmix** [comment]
this is part of #village-tracker !

File: docs/throughs/take.md [hunk 1]
```diff
@@ -1,13 +1,55 @@
 # pull-stream/throughs/take
 
-## usage
+## Example usage
 
-### `take = require('pull-stream/throughs/take')`
+```js
+var pull = require('pull-stream')
+// var take = require('pull-stream/throughs/take') // if you just need take
```
Comment (harrysarson) [inline docs/throughs/take.md]: Does this line need to be commented out?

File: docs/throughs/take.md [hunk 2]
```diff
@@ -1,13 +1,55 @@
...
+    console.log(results)
+    // => [1, 2, 3, 4, 5]
+  })
+})
+```
+
+```js
+pull(
+  pull.values([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]),
+  pull.take(n => n < 4.6), { last: false }),  // exclude the last value test (5)
```
Comment (harrysarson) [inline docs/throughs/take.md]:
```suggestion
  pull.take(n => n < 4.6), { last: false }),  // exclude the last value tested (5)
```

Comment (ahdinosaur) [inline docs/throughs/take.md]: maybe here don't use `last: false` (but keep the comment) to show that by default it's false.

File: docs/throughs/take.md (hunk 1 above)
Comment (mixmix) [reply docs/throughs/take.md]: yep, it's not used. I'm showing an alternative.

This PR is of the class of "it's better than it was", so I'm not going to tune this particular thing. Thanks for input though :)

File: docs/throughs/take.md (hunk 2 above)
Comment (mixmix) [reply docs/throughs/take.md]: perfect, thank you @harrysarson

Comment (mixmix) [reply docs/throughs/take.md]: could do, I'm going to leave it as is, because using this `testFn` isn't used much as far as I know.
I think they should be seperated into : 
- `pull.take(n)`
- `pull.till(testFn, opts)`

but that's another PR

**Comment by mixmix** [comment]
@ahdinosaur , I've merged this. Does it need publishing to trigger the aggregator docs or anything?

**Comment by dominictarr** [comment]
published as 3.6.11

**Comment by mixmix** [comment]
dominic executing his new publishing priviledges!
